packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -32,14 +32,19 @@ const NavLogo = ({
   isCard,
   ...customProps
 }) => {
-  const image = <img className={cx('img')} src={url} alt={altText} />;
+  let logoContent;
+  if (url) {
+    logoContent = <img className={cx('img')} src={url} alt={altText} />;
+  } else {
+    logoContent = (<p className={cx('logo-text')}>{altText}</p>);
```

Comment:
the purpose of the altText is that it is displayed when no url is provided or the image fails to load, this shouldnt be needed.

Reply:
make sense I added the required styles to img.